parquet-hadoop/src/main/java/org/apache/parquet/ParquetReadOptions.java

```diff
@@ -223,6 +224,13 @@ public Builder withRecordFilter(FilterCompat.Filter rowGroupFilter) {
       return this;
     }
 
+    public Builder useDirectByteBufferAllocator(boolean val) {
```

Comment:
```suggestion
    public Builder useDirectByteBufferAllocator(boolean allocator) {
```

Reply:
Thanks for replies. 

In `Spark`,   [ParquetFileReader](https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFileFormat.scala#L272)  passes `options` from `Hadoop Configuration` (using ParquetReadOptions.Builder) rather than `ParquetReadOptions`.

In `ParquetReadOptions.Builder` constructor,  now `Allocator` param cannot be passed to `ParquetReadOptions`.

Reply:
Makes sense, could you still rename `val` to `useDirectByteBuffer`?

Reply:
Hi~ I have fixed this and rebased the newest codes.

Can you review this again?